### ISSUE TITLE
ARIA landmarks review 

### ISSUE THREAD

**Issue by ajguerr4** [gios-asu/ASU-Web-Standards-Wordpress-Theme#348]
In our Theme we have landmarks for our pages and there are a couple that need to be reviewed to see if they meet standards. 

The first has to do with hero images. 
Most of our hero images rest in custom fields. Im getting warnings from SiteImprove for them being potentially inaccessible. They are labeled as just <sections. These warnings are given on every page with hero images. And separately they have a second warning asking if they could be labeled more appropriately.

This warning is given for hero image sections.
Warning: [The page contains an HTML5 <section or a WAI-ARIA landmark of the type 'region'. These types of landmarks are very general and should be named to help users identify the purpose of the landmark.] 
they should probably be banner roles in the header tag. But I may be wrong. 

The Second landmark warning to be assessed is the article landmark. SiteImprove pulls up the article landmark on every page and asks if it's labeled correctly. In most of those cases the "main" landmark and the "article" landmark are highlighted for the same amount content. They probably should be different.

Here are websites for sources into this. The first source is offered by SiteImprove as a resource. The second is from my own research into what roles to use for the hero images.

ARIA11: https://www.w3.org/TR/WCAG20-TECHS/ARIA11.html
ARIA Roles 101: http://www.webteacher.ws/2010/10/14/aria-roles-101/

This picture shows what the warning appears as.
![screen shot 2018-05-09 at 10 33 20 am png](https://user-images.githubusercontent.com/38890781/39829816-75f1767a-5374-11e8-9a7f-1c5d94128ab0.png)
